content/jobs/melee/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+menu:
+  main:
+    name: Melee
+    parent: jobs
+role: healer
```

Comment:
Roles are not in plural, but I assume that's part of the other PR as well.

Reply:
Yes, anything that's a frontmatter change should be addressed as we fix the editor field(s)